org.osgi.test.common/src/main/java/org/osgi/test/common/annotation/config/Configurations.java

```diff
@@ -0,0 +1,23 @@
+package org.osgi.test.common.annotation.config;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Inherited
+@Target({
+	TYPE, METHOD, PARAMETER, FIELD
+})
+@Retention(RUNTIME)
+@Documented
+public @interface Configurations {
```

Comment:
I've not had much experience practically with defining repeatable annotations, but I just found this little tip which I thought might be useful and help tidy up the code a little - basically, to make the container annotation an inner class:

https://dzone.com/articles/repeatable-annotations-in-java-8-1

Reply:
I don't find the idea of an inner type that compelling. We did not do this for repeatable OSGi annotations.

Reply:
I defer to the (more experienced) majority.